modules/ROOT/pages/ManualInstallDirections.adoc

```diff
@@ -69,3 +69,39 @@ But you can try to name your files so its name comes after the name of the depen
 If you need to know how to enable `developmentMode`,
 please refer to the xref:SMLConfiguration.adoc[SML Configuration Page].
 ====
+
+== Installing mods on Linux (Steam) as a player - SML version 2.2.1
+
+[WARNING]
+====
+_THIS IS UNSUPPORTED -> NO SUPPORT SHOULD BE EXPECTED FOR ANYONE ATTEMPTING THIS_.
+====
+
+. `cd <game root>/FactoryGame/Binaries/Win64`
+. Navigate to https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/tag/v2.0.11 and download into the `./Win64` folder:
```

Comment:
Please change this to use a `latest` url (https://github.com/satisfactorymodding/SatisfactoryModBootstrapper/releases/latest)

Reply:
Is it preferable to ensure backwards compatibility (keep it as is), or add the responsibility to update/correct documentation if anything ever changes?

Would be nice to hear from maintainers (you maybe?) where you'd like to draw that line before I change it =)
Don't feel strongly about it myself, would like to give the choice to the majority of the maintainers.

Reply:
@Vilsol ---^

Reply:
The docs aren't always updated when new SML updates release, so I would rather have it as a `latest` link. That way we won't have to update it with every release. If there's a problem and a certain old release has to be used, we would ideally put a note about that here.

Reply:
Aight, that's two for, and no against in as many days! Change incoming as soon as I get a chance!

Reply:
Done! =)